Dockerfile

```diff
@@ -1,12 +1,16 @@
 FROM registry.access.redhat.com/ubi8/ubi-minimal:8.6
 
+COPY --from=quay.io/eerez/python-builder:ubi-minimal8.6-3.11.0 /python /python-installation
```

Comment:
What happens if we upgrade to ubi9 ? Would this image still work as-is?

Reply:
We can use ubi9 image for that - quay.io/eerez/python-builder:ubi-minimal9.0.0-1687-3.11.0

Reply:
Isn't updating this reference each base-image upgrade this a bit cumbersome? Why do we need the python 3.11? Is the version packaged with our base image is not enough?

Reply:
There is some security issues on previous versions so I tried to update to the latest. But because of the CI fails I'll just update to the last version of python3.9